Synthesis failed for java-analytics-data

Hello! Autosynth couldn't regenerate java-analytics-data. :broken_heart:

Here's the output from running `synth.py`:

```
ynth [DEBUG] > Running: git log -1 --pretty=%at e28aa8570dc86136fe677202c976f4ee00d77f61
2021-02-11 15:07:07,186 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6de7abea4c424025bfcf996ef40c96e53c070100
2021-02-11 15:07:07,189 autosynth [DEBUG] > Running: git log -1 --pretty=%at 614b4fc8969008b086fbd6d3d99dfdb6fcf0a4a1
2021-02-11 15:07:07,192 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8761f14a2a2a2cde7e44c9bf8a5e901273a71812
2021-02-11 15:07:07,195 autosynth [DEBUG] > Running: git log -1 --pretty=%at 66e0de02e649c5b90f873836bdbc97585d91ed46
2021-02-11 15:07:07,197 autosynth [DEBUG] > Running: git log -1 --pretty=%at dd89b0cc1bffc39d1fb5c91c6c2bb3532f237792
2021-02-11 15:07:07,200 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1e0c7413684ca6f6322620ecfc0d3e0352933dc1
2021-02-11 15:07:07,203 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-11 15:07:07,206 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-11 15:07:07,209 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-11 15:07:07,212 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-11 15:07:07,215 autosynth [DEBUG] > Running: git checkout 545ccbc1df86f50a64e0ad16428d5f6490ef9dde
Note: checking out '545ccbc1df86f50a64e0ad16428d5f6490ef9dde'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-11 15:07:07,233 autosynth [DEBUG] > Running: git checkout f3a60f63c13fb434745ea59b990a82d6ffc803b5
Note: checking out 'f3a60f63c13fb434745ea59b990a82d6ffc803b5'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at f3a60f63 docs: Update rules for currency_code in budget_amount.
2021-02-11 15:07:07,247 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-11 15:07:07,252 autosynth [DEBUG] > Running: git branch -f autosynth-16
2021-02-11 15:07:07,255 autosynth [DEBUG] > Running: git checkout autosynth-16
Switched to branch 'autosynth-16'
2021-02-11 15:07:07,259 autosynth [INFO] > Running synthtool
2021-02-11 15:07:07,259 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-11 15:07:07,260 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-data/16/sponge_log.log
2021-02-11 15:07:07,261 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-11 15:07:07,492 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-data/synth.py.
On branch autosynth-16
nothing to commit, working tree clean
2021-02-11 15:07:07,632 synthtool [DEBUG] > Ensuring dependencies.
2021-02-11 15:07:07,642 synthtool [DEBUG] > Cloning googleapis.
2021-02-11 15:07:07,643 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-11 15:07:07,647 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-11 15:07:07,651 synthtool [DEBUG] > Generating code for: //google/analytics/data/v1alpha:google-analytics-data-v1alpha-java.
2021-02-11 15:07:11,770 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpoqi7w4l7.
2021-02-11 15:07:11,771 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpoqi7w4l7/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-11 15:07:11,775 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpoqi7w4l7/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmpoqi7w4l7/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-11 15:07:11,775 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpoqi7w4l7/google-cloud-analytics-data-v1alpha-java/grpc-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-data/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmpoqi7w4l7/google-cloud-analytics-data-v1alpha-java/gapic-google-analytics-data-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-11 15:07:11,826 autosynth [ERROR] > Synthesis failed
2021-02-11 15:07:11,827 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-11 15:07:11,832 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-11 15:07:11,836 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/082d9d81-121c-4af4-9cc3-cc51dcb2053d/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-data;failed=false).

Autosynth is still having trouble generating java-analytics-data. :sob:

Here's the output from running `synth.py`:

```
06:43,559 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6de7abea4c424025bfcf996ef40c96e53c070100
2021-02-12 02:06:43,563 autosynth [DEBUG] > Running: git log -1 --pretty=%at 614b4fc8969008b086fbd6d3d99dfdb6fcf0a4a1
2021-02-12 02:06:43,567 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8761f14a2a2a2cde7e44c9bf8a5e901273a71812
2021-02-12 02:06:43,571 autosynth [DEBUG] > Running: git log -1 --pretty=%at 66e0de02e649c5b90f873836bdbc97585d91ed46
2021-02-12 02:06:43,575 autosynth [DEBUG] > Running: git log -1 --pretty=%at dd89b0cc1bffc39d1fb5c91c6c2bb3532f237792
2021-02-12 02:06:43,578 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1e0c7413684ca6f6322620ecfc0d3e0352933dc1
2021-02-12 02:06:43,582 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-12 02:06:43,586 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-12 02:06:43,590 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-12 02:06:43,594 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-12 02:06:43,597 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-12 02:06:43,601 autosynth [DEBUG] > Running: git checkout 545ccbc1df86f50a64e0ad16428d5f6490ef9dde
Note: checking out '545ccbc1df86f50a64e0ad16428d5f6490ef9dde'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-12 02:06:43,620 autosynth [DEBUG] > Running: git checkout c059c46957b0eb83078e61b4a5170c3d09ea96e6
Note: checking out 'c059c46957b0eb83078e61b4a5170c3d09ea96e6'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at c059c469 chore: Operations proto comment cleanup
2021-02-12 02:06:43,637 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-12 02:06:43,644 autosynth [DEBUG] > Running: git branch -f autosynth-17
2021-02-12 02:06:43,648 autosynth [DEBUG] > Running: git checkout autosynth-17
Switched to branch 'autosynth-17'
2021-02-12 02:06:43,653 autosynth [INFO] > Running synthtool
2021-02-12 02:06:43,654 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-12 02:06:43,654 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-data/17/sponge_log.log
2021-02-12 02:06:43,656 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-12 02:06:43,911 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-data/synth.py.
On branch autosynth-17
nothing to commit, working tree clean
2021-02-12 02:06:44,068 synthtool [DEBUG] > Ensuring dependencies.
2021-02-12 02:06:44,083 synthtool [DEBUG] > Cloning googleapis.
2021-02-12 02:06:44,084 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-12 02:06:44,090 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-12 02:06:44,094 synthtool [DEBUG] > Generating code for: //google/analytics/data/v1alpha:google-analytics-data-v1alpha-java.
2021-02-12 02:06:48,202 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpj2ljqzug.
2021-02-12 02:06:48,204 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpj2ljqzug/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-12 02:06:48,208 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpj2ljqzug/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmpj2ljqzug/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-12 02:06:48,209 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpj2ljqzug/google-cloud-analytics-data-v1alpha-java/grpc-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-data/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmpj2ljqzug/google-cloud-analytics-data-v1alpha-java/gapic-google-analytics-data-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-12 02:06:48,269 autosynth [ERROR] > Synthesis failed
2021-02-12 02:06:48,269 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-12 02:06:48,275 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-12 02:06:48,281 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/68f2f6b8-7431-47a6-a634-59c5352f7382/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-data;failed=false).

Autosynth is still having trouble generating java-analytics-data. :sob:

Here's the output from running `synth.py`:

```
autosynth [DEBUG] > Running: git log -1 --pretty=%at 1e0c7413684ca6f6322620ecfc0d3e0352933dc1
2021-02-12 15:07:50,424 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-12 15:07:50,427 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-12 15:07:50,430 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-12 15:07:50,433 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-12 15:07:50,437 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-12 15:07:50,440 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-12 15:07:50,443 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-12 15:07:50,446 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-12 15:07:50,449 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-12 15:07:50,452 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-12 15:07:50,456 autosynth [DEBUG] > Running: git checkout 545ccbc1df86f50a64e0ad16428d5f6490ef9dde
Note: checking out '545ccbc1df86f50a64e0ad16428d5f6490ef9dde'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-12 15:07:50,475 autosynth [DEBUG] > Running: git checkout 178d16d9f99ee85ffa65dff304f40324c35d0668
Note: checking out '178d16d9f99ee85ffa65dff304f40324c35d0668'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 178d16d9 chore(bazel): update Go microgenerator to v0.18.1
2021-02-12 15:07:50,492 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-12 15:07:50,499 autosynth [DEBUG] > Running: git branch -f autosynth-22
2021-02-12 15:07:50,502 autosynth [DEBUG] > Running: git checkout autosynth-22
Switched to branch 'autosynth-22'
2021-02-12 15:07:50,507 autosynth [INFO] > Running synthtool
2021-02-12 15:07:50,507 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-12 15:07:50,507 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-data/22/sponge_log.log
2021-02-12 15:07:50,509 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-12 15:07:50,762 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-data/synth.py.
On branch autosynth-22
nothing to commit, working tree clean
2021-02-12 15:07:50,911 synthtool [DEBUG] > Ensuring dependencies.
2021-02-12 15:07:50,922 synthtool [DEBUG] > Cloning googleapis.
2021-02-12 15:07:50,923 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-12 15:07:50,928 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-12 15:07:50,931 synthtool [DEBUG] > Generating code for: //google/analytics/data/v1alpha:google-analytics-data-v1alpha-java.
2021-02-12 15:07:55,119 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpgp7404rc.
2021-02-12 15:07:55,121 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpgp7404rc/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-12 15:07:55,124 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpgp7404rc/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmpgp7404rc/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-12 15:07:55,125 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpgp7404rc/google-cloud-analytics-data-v1alpha-java/grpc-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-data/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmpgp7404rc/google-cloud-analytics-data-v1alpha-java/gapic-google-analytics-data-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-12 15:07:55,174 autosynth [ERROR] > Synthesis failed
2021-02-12 15:07:55,175 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-12 15:07:55,181 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-12 15:07:55,186 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/5e3736d6-6ebd-480c-bf62-78413733613e/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-data;failed=false).

Autosynth is still having trouble generating java-analytics-data. :sob:

Here's the output from running `synth.py`:

```
UG] > Running: git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-13 02:04:41,356 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-13 02:04:41,359 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-13 02:04:41,362 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-13 02:04:41,366 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-13 02:04:41,369 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-13 02:04:41,372 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-13 02:04:41,375 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-13 02:04:41,378 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-13 02:04:41,382 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-13 02:04:41,385 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-13 02:04:41,388 autosynth [DEBUG] > Running: git checkout 545ccbc1df86f50a64e0ad16428d5f6490ef9dde
Note: checking out '545ccbc1df86f50a64e0ad16428d5f6490ef9dde'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-13 02:04:41,407 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-13 02:04:41,424 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-13 02:04:41,430 autosynth [DEBUG] > Running: git branch -f autosynth-23
2021-02-13 02:04:41,434 autosynth [DEBUG] > Running: git checkout autosynth-23
Switched to branch 'autosynth-23'
2021-02-13 02:04:41,439 autosynth [INFO] > Running synthtool
2021-02-13 02:04:41,439 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-13 02:04:41,439 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-data/23/sponge_log.log
2021-02-13 02:04:41,442 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-13 02:04:41,678 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-data/synth.py.
On branch autosynth-23
nothing to commit, working tree clean
2021-02-13 02:04:41,824 synthtool [DEBUG] > Ensuring dependencies.
2021-02-13 02:04:41,836 synthtool [DEBUG] > Cloning googleapis.
2021-02-13 02:04:41,837 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-13 02:04:41,842 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-13 02:04:41,846 synthtool [DEBUG] > Generating code for: //google/analytics/data/v1alpha:google-analytics-data-v1alpha-java.
2021-02-13 02:04:45,809 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpy3esuy9g.
2021-02-13 02:04:45,811 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpy3esuy9g/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-13 02:04:45,814 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpy3esuy9g/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmpy3esuy9g/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-13 02:04:45,815 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpy3esuy9g/google-cloud-analytics-data-v1alpha-java/grpc-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-data/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmpy3esuy9g/google-cloud-analytics-data-v1alpha-java/gapic-google-analytics-data-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-13 02:04:45,861 autosynth [ERROR] > Synthesis failed
2021-02-13 02:04:45,861 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-13 02:04:45,867 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-13 02:04:45,872 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/562f66c4-78e1-4e49-814f-ad1fe2f28808/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-data;failed=false).

Autosynth is still having trouble generating java-analytics-data. :sob:

Here's the output from running `synth.py`:

```
UG] > Running: git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-13 15:06:17,396 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-13 15:06:17,399 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-13 15:06:17,402 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-13 15:06:17,405 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-13 15:06:17,408 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-13 15:06:17,410 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-13 15:06:17,413 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-13 15:06:17,416 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-13 15:06:17,419 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-13 15:06:17,422 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-13 15:06:17,425 autosynth [DEBUG] > Running: git checkout 545ccbc1df86f50a64e0ad16428d5f6490ef9dde
Note: checking out '545ccbc1df86f50a64e0ad16428d5f6490ef9dde'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-13 15:06:17,444 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-13 15:06:17,459 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-13 15:06:17,466 autosynth [DEBUG] > Running: git branch -f autosynth-23
2021-02-13 15:06:17,469 autosynth [DEBUG] > Running: git checkout autosynth-23
Switched to branch 'autosynth-23'
2021-02-13 15:06:17,474 autosynth [INFO] > Running synthtool
2021-02-13 15:06:17,474 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-13 15:06:17,474 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-data/23/sponge_log.log
2021-02-13 15:06:17,476 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-13 15:06:17,727 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-data/synth.py.
On branch autosynth-23
nothing to commit, working tree clean
2021-02-13 15:06:17,873 synthtool [DEBUG] > Ensuring dependencies.
2021-02-13 15:06:17,884 synthtool [DEBUG] > Cloning googleapis.
2021-02-13 15:06:17,885 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-13 15:06:17,889 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-13 15:06:17,893 synthtool [DEBUG] > Generating code for: //google/analytics/data/v1alpha:google-analytics-data-v1alpha-java.
2021-02-13 15:06:21,908 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp_72snk_g.
2021-02-13 15:06:21,911 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp_72snk_g/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-13 15:06:21,918 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp_72snk_g/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmp_72snk_g/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-13 15:06:21,919 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp_72snk_g/google-cloud-analytics-data-v1alpha-java/grpc-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-data/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmp_72snk_g/google-cloud-analytics-data-v1alpha-java/gapic-google-analytics-data-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-13 15:06:21,989 autosynth [ERROR] > Synthesis failed
2021-02-13 15:06:21,989 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-13 15:06:21,995 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-13 15:06:21,999 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/2f1022c1-09f4-4cc1-bac8-062aa7db90e2/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-data;failed=false).

Autosynth is still having trouble generating java-analytics-data. :sob:

Here's the output from running `synth.py`:

```
UG] > Running: git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-14 02:03:56,803 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-14 02:03:56,806 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-14 02:03:56,809 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-14 02:03:56,812 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-14 02:03:56,815 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-14 02:03:56,818 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-14 02:03:56,821 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-14 02:03:56,824 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-14 02:03:56,826 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-14 02:03:56,829 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-14 02:03:56,833 autosynth [DEBUG] > Running: git checkout 545ccbc1df86f50a64e0ad16428d5f6490ef9dde
Note: checking out '545ccbc1df86f50a64e0ad16428d5f6490ef9dde'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-14 02:03:56,850 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-14 02:03:56,865 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-14 02:03:56,870 autosynth [DEBUG] > Running: git branch -f autosynth-23
2021-02-14 02:03:56,874 autosynth [DEBUG] > Running: git checkout autosynth-23
Switched to branch 'autosynth-23'
2021-02-14 02:03:56,878 autosynth [INFO] > Running synthtool
2021-02-14 02:03:56,878 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-14 02:03:56,878 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-data/23/sponge_log.log
2021-02-14 02:03:56,880 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-14 02:03:57,104 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-data/synth.py.
On branch autosynth-23
nothing to commit, working tree clean
2021-02-14 02:03:57,246 synthtool [DEBUG] > Ensuring dependencies.
2021-02-14 02:03:57,257 synthtool [DEBUG] > Cloning googleapis.
2021-02-14 02:03:57,258 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-14 02:03:57,262 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-14 02:03:57,266 synthtool [DEBUG] > Generating code for: //google/analytics/data/v1alpha:google-analytics-data-v1alpha-java.
2021-02-14 02:04:01,109 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp40sjhfri.
2021-02-14 02:04:01,110 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp40sjhfri/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-14 02:04:01,114 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp40sjhfri/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmp40sjhfri/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-14 02:04:01,114 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp40sjhfri/google-cloud-analytics-data-v1alpha-java/grpc-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-data/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmp40sjhfri/google-cloud-analytics-data-v1alpha-java/gapic-google-analytics-data-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-14 02:04:01,173 autosynth [ERROR] > Synthesis failed
2021-02-14 02:04:01,173 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-14 02:04:01,178 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-14 02:04:01,183 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/31fd9052-93c2-4dbc-ae81-21c9951f8f8f/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-data;failed=false).

Autosynth is still having trouble generating java-analytics-data. :sob:

Here's the output from running `synth.py`:

```
UG] > Running: git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-14 15:04:35,637 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-14 15:04:35,640 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-14 15:04:35,643 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-14 15:04:35,647 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-14 15:04:35,650 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-14 15:04:35,653 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-14 15:04:35,657 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-14 15:04:35,660 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-14 15:04:35,663 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-14 15:04:35,667 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-14 15:04:35,671 autosynth [DEBUG] > Running: git checkout 545ccbc1df86f50a64e0ad16428d5f6490ef9dde
Note: checking out '545ccbc1df86f50a64e0ad16428d5f6490ef9dde'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-14 15:04:35,689 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-14 15:04:35,705 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-14 15:04:35,711 autosynth [DEBUG] > Running: git branch -f autosynth-23
2021-02-14 15:04:35,715 autosynth [DEBUG] > Running: git checkout autosynth-23
Switched to branch 'autosynth-23'
2021-02-14 15:04:35,721 autosynth [INFO] > Running synthtool
2021-02-14 15:04:35,721 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-14 15:04:35,721 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-data/23/sponge_log.log
2021-02-14 15:04:35,723 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-14 15:04:35,966 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-data/synth.py.
On branch autosynth-23
nothing to commit, working tree clean
2021-02-14 15:04:36,120 synthtool [DEBUG] > Ensuring dependencies.
2021-02-14 15:04:36,132 synthtool [DEBUG] > Cloning googleapis.
2021-02-14 15:04:36,133 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-14 15:04:36,137 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-14 15:04:36,141 synthtool [DEBUG] > Generating code for: //google/analytics/data/v1alpha:google-analytics-data-v1alpha-java.
2021-02-14 15:04:40,210 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmphf0uh5ti.
2021-02-14 15:04:40,211 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmphf0uh5ti/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-14 15:04:40,215 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmphf0uh5ti/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmphf0uh5ti/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-14 15:04:40,216 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmphf0uh5ti/google-cloud-analytics-data-v1alpha-java/grpc-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-data/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmphf0uh5ti/google-cloud-analytics-data-v1alpha-java/gapic-google-analytics-data-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-14 15:04:40,289 autosynth [ERROR] > Synthesis failed
2021-02-14 15:04:40,289 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-14 15:04:40,295 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-14 15:04:40,300 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/9a734575-adfb-43a1-acd6-2d5255cd7476/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-data;failed=false).

Autosynth is still having trouble generating java-analytics-data. :sob:

Here's the output from running `synth.py`:

```
UG] > Running: git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-15 02:05:51,860 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-15 02:05:51,864 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-15 02:05:51,868 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-15 02:05:51,872 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-15 02:05:51,875 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-15 02:05:51,879 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-15 02:05:51,884 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-15 02:05:51,887 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-15 02:05:51,891 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-15 02:05:51,895 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-15 02:05:51,899 autosynth [DEBUG] > Running: git checkout 545ccbc1df86f50a64e0ad16428d5f6490ef9dde
Note: checking out '545ccbc1df86f50a64e0ad16428d5f6490ef9dde'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-15 02:05:51,919 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-15 02:05:51,935 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-15 02:05:51,942 autosynth [DEBUG] > Running: git branch -f autosynth-23
2021-02-15 02:05:51,946 autosynth [DEBUG] > Running: git checkout autosynth-23
Switched to branch 'autosynth-23'
2021-02-15 02:05:51,952 autosynth [INFO] > Running synthtool
2021-02-15 02:05:51,952 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-15 02:05:51,952 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-data/23/sponge_log.log
2021-02-15 02:05:51,955 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-15 02:05:52,204 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-data/synth.py.
On branch autosynth-23
nothing to commit, working tree clean
2021-02-15 02:05:52,356 synthtool [DEBUG] > Ensuring dependencies.
2021-02-15 02:05:52,369 synthtool [DEBUG] > Cloning googleapis.
2021-02-15 02:05:52,370 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-15 02:05:52,376 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-15 02:05:52,381 synthtool [DEBUG] > Generating code for: //google/analytics/data/v1alpha:google-analytics-data-v1alpha-java.
2021-02-15 02:05:56,433 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpkzbyzwr6.
2021-02-15 02:05:56,434 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpkzbyzwr6/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-15 02:05:56,438 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpkzbyzwr6/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmpkzbyzwr6/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-15 02:05:56,439 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpkzbyzwr6/google-cloud-analytics-data-v1alpha-java/grpc-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-data/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmpkzbyzwr6/google-cloud-analytics-data-v1alpha-java/gapic-google-analytics-data-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-15 02:05:56,505 autosynth [ERROR] > Synthesis failed
2021-02-15 02:05:56,506 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-15 02:05:56,511 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-15 02:05:56,517 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f88c5505-c115-4c44-a4df-7518606354ad/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-data;failed=false).

Autosynth is still having trouble generating java-analytics-data. :sob:

Here's the output from running `synth.py`:

```
UG] > Running: git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-15 15:06:01,304 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-15 15:06:01,308 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-15 15:06:01,312 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-15 15:06:01,317 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-15 15:06:01,321 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-15 15:06:01,325 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-15 15:06:01,329 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-15 15:06:01,333 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-15 15:06:01,338 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-15 15:06:01,342 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-15 15:06:01,347 autosynth [DEBUG] > Running: git checkout 545ccbc1df86f50a64e0ad16428d5f6490ef9dde
Note: checking out '545ccbc1df86f50a64e0ad16428d5f6490ef9dde'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-15 15:06:01,367 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-15 15:06:01,385 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-15 15:06:01,392 autosynth [DEBUG] > Running: git branch -f autosynth-23
2021-02-15 15:06:01,396 autosynth [DEBUG] > Running: git checkout autosynth-23
Switched to branch 'autosynth-23'
2021-02-15 15:06:01,402 autosynth [INFO] > Running synthtool
2021-02-15 15:06:01,402 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-15 15:06:01,402 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-data/23/sponge_log.log
2021-02-15 15:06:01,404 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-15 15:06:01,655 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-data/synth.py.
On branch autosynth-23
nothing to commit, working tree clean
2021-02-15 15:06:01,806 synthtool [DEBUG] > Ensuring dependencies.
2021-02-15 15:06:01,820 synthtool [DEBUG] > Cloning googleapis.
2021-02-15 15:06:01,821 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-15 15:06:01,826 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-15 15:06:01,831 synthtool [DEBUG] > Generating code for: //google/analytics/data/v1alpha:google-analytics-data-v1alpha-java.
2021-02-15 15:06:05,830 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp5tkirhdk.
2021-02-15 15:06:05,831 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp5tkirhdk/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-15 15:06:05,835 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp5tkirhdk/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmp5tkirhdk/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-15 15:06:05,836 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp5tkirhdk/google-cloud-analytics-data-v1alpha-java/grpc-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-data/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmp5tkirhdk/google-cloud-analytics-data-v1alpha-java/gapic-google-analytics-data-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-15 15:06:05,887 autosynth [ERROR] > Synthesis failed
2021-02-15 15:06:05,887 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-15 15:06:05,893 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-15 15:06:05,899 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/054c0893-56c7-417a-9b9b-8974285e2450/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-data;failed=false).

Autosynth is still having trouble generating java-analytics-data. :sob:

Here's the output from running `synth.py`:

```
UG] > Running: git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-16 02:03:50,315 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-16 02:03:50,318 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-16 02:03:50,321 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-16 02:03:50,323 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-16 02:03:50,326 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-16 02:03:50,329 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-16 02:03:50,332 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-16 02:03:50,335 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-16 02:03:50,338 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-16 02:03:50,341 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-16 02:03:50,344 autosynth [DEBUG] > Running: git checkout 545ccbc1df86f50a64e0ad16428d5f6490ef9dde
Note: checking out '545ccbc1df86f50a64e0ad16428d5f6490ef9dde'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-16 02:03:50,362 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-16 02:03:50,376 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-16 02:03:50,382 autosynth [DEBUG] > Running: git branch -f autosynth-23
2021-02-16 02:03:50,385 autosynth [DEBUG] > Running: git checkout autosynth-23
Switched to branch 'autosynth-23'
2021-02-16 02:03:50,390 autosynth [INFO] > Running synthtool
2021-02-16 02:03:50,390 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-16 02:03:50,390 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-data/23/sponge_log.log
2021-02-16 02:03:50,392 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-16 02:03:50,615 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-data/synth.py.
On branch autosynth-23
nothing to commit, working tree clean
2021-02-16 02:03:50,755 synthtool [DEBUG] > Ensuring dependencies.
2021-02-16 02:03:50,766 synthtool [DEBUG] > Cloning googleapis.
2021-02-16 02:03:50,766 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-16 02:03:50,771 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-16 02:03:50,774 synthtool [DEBUG] > Generating code for: //google/analytics/data/v1alpha:google-analytics-data-v1alpha-java.
2021-02-16 02:03:54,480 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp0_ahkpzl.
2021-02-16 02:03:54,482 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp0_ahkpzl/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-16 02:03:54,485 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp0_ahkpzl/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmp0_ahkpzl/google-cloud-analytics-data-v1alpha-java/proto-google-analytics-data-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-16 02:03:54,486 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp0_ahkpzl/google-cloud-analytics-data-v1alpha-java/grpc-google-analytics-data-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-data/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmp0_ahkpzl/google-cloud-analytics-data-v1alpha-java/gapic-google-analytics-data-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-16 02:03:54,541 autosynth [ERROR] > Synthesis failed
2021-02-16 02:03:54,541 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 545ccbc build(java): update autorelease script (#137)
2021-02-16 02:03:54,547 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-16 02:03:54,551 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/0932c387-5458-427c-821a-c508d306b680/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-data;failed=false).

--- a/synth.py
+++ b/synth.py
@@ -14,20 +14,17 @@
 
 """This script is used to synthesize generated parts of this library."""
 
-import synthtool as s
-import synthtool.gcp as gcp
 import synthtool.languages.java as java
 
 service = 'analytics-data'
 versions = ['v1alpha']
 
 for version in versions:
-  java.bazel_library(
-      service=service,
-      version=version,
-      proto_path=f'google/analytics/data/{version}',
-      bazel_target=f'//google/analytics/data/{version}:google-analytics-data-{version}-java',
-      cloud_api=False,
+  java.pregenerated_library(
+    service=service,
+    version=version,
+    path=f'google/analytics/data/{version}',
+    cloud_api=False,
   )
 
 java.common_templates()

